kolibri/content/fixtures/content_import_test.json

```diff
@@ -0,0 +1,356 @@
+[
```

Comment:
Duplicated this here, so as to have a unique fixture for the content_import tests, so that we don't have to update the schema reflection data jsons just because the base `content_test.json` is updated (as it has been by @ralphiee22 on 0.4.x).